index.js

```diff
@@ -82,9 +82,15 @@ function fetchPageOfDataAndFilter(url) {
                     })
                     .catch((err) => {
                         console.log('Error turning response into JSON:', err);
+
+                        return reject(err);
                     });
             })
-            .catch((err) => console.log('ERROR GRABBING INFO FROM GITHUB!', err));
+            .catch((err) => {
+                console.log('ERROR GRABBING INFO FROM GITHUB!', err);
+
+                return reject(err);
```

Comment:
Same questions here, about reject vs throw, and whether the requirement is to error or log and continue processing.

Reply:
Unless I'm mistaken, I don't understand how adding these `reject` calls changes what this function does other than to prevent the returned promise from ever hanging indefinitely.

I can remove the promise rejections if they do pose a problem. This means I'll have to comment out or remove the tests that check for errors though since unhandled promises aren't testable with mocha.

Reply:
Same here. Erroring and stopping is what I want it to do since the user will have to re-run the whole process anyway if it errors finding one of the users.